### PR TITLE
Some quickstart instructions.

### PR DESCRIPTION
**Pull request by zck**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,24 @@
...
+
+Run `M-x swb-connection-mysql`. It will prompt for `host`, `port`, `user`, `password`, and the database inside the MySQL server to use.
+
+The workbench buffer will open.
+
+## Using the workbench buffer
+
+The workbench buffer is the sql-workbench's main interface to the database. Here are some things you can do with it.
+
+* Write SQL statements in the buffer, then with point inside the sql statement, run `swb-send-current-query` (`C-c C-c`) to run that query on the database and send the
```
Comment (Fuco1) [inline README.md]: Seems like you cut the sentence in half here, or maybe github is going crazy with the rendering? :O

File: README.md [hunk 2]
```diff
@@ -2,6 +2,24 @@
...
+
+The workbench buffer will open.
+
+## Using the workbench buffer
+
+The workbench buffer is the sql-workbench's main interface to the database. Here are some things you can do with it.
+
+* Write SQL statements in the buffer, then with point inside the sql statement, run `swb-send-current-query` (`C-c C-c`) to run that query on the database and send the 
+
+* `swb-describe-table` (C-c C-t) prompts for a table name, and displays the table definition.
```
Comment (Fuco1) [inline README.md]: There's also `C-c C-d` which works the same (prompts for table name) and shows the data (first 500 entries) it's quite useful to quickly look around, we might want to add that too

**Comment by Fuco1** [comment]
Thanks for the PR!  Much appreciated.  There are some minor things to address but then we can go ahead and merge :) I'll try to elaborate some more.

It's really motivating to see people come and do this work so thank you again :)

**Comment by zck** [comment]
I completed the incomplete sentence, added `#'swb-show-data-in-table`, and reworded `#'swb-send-current-query` so that all three high-level functions have their descriptions formatted the same (with the name and keybinding first, then the description).

I'm glad you're open to this; it was certainly better than having to write my own -- which was the alternate solution.

**Comment by Fuco1** [comment]
@zck You wrote

> Run `M-x swb-connection-mysql`. It will prompt for `host`, `port`, `user`, `password`, and the database inside the MySQL server to use.

I don't have such function.  Did you write that yourself?  If so it would be super cool to have it merged upstream! :D

I haven't even tried that because I always store the "credentials" in the workbench file (C-c C-s) and then it just reconnects automatically when I send the query for the first time).

**Comment by zck** [comment]
Ugh, you're right, that isn't a function. I was thinking of `swb-new-mysql-workbench`. Isn't that the right way to start the workbench?

If that's right, I fixed it in this PR: https://github.com/Fuco1/sql-workbench/pull/20. Sorry about that.

Also, what's the point of storing the credentials in the workbench? Is it to save the workbench as a file, then re-open it later? I don't seem to need to do that if I run `swb-new-mysql-workbench`, then do anything inside that workbench file.

**Comment by Fuco1** [comment]
@zck I honestly don't ever remember writing that function :D

I always save the config in the file and then reopen it later and it knows where to connect.  Usually in each of my projects I have a file `mysql.swb` (I associate swb with this mode) and when I want to do anything with the database I open that and I hve all my previous work + connection details so I literally just have to `C-c C-c` and type in the password.  It's super convenient.

**Pull request merged**